docs/_toc.yml

```diff
@@ -1,6 +1,7 @@
-- file: intro
-
-- part: Tutorials
+format: jb-book
+root: intro
+parts:
+- caption: Tutorials
```

Comment:
Maybe we wanna call this `title:` rather than `caption:`? I know that Sphinx calls it caption but I feel like "title" will be more intuitive for people

Reply:
eurgh maybe, but it will mean additonal code/complexity needs to be implemented in `sphinx-external-toc`, to map toctree options to different keys

Reply:
Ah i thought it would already work, in that case let's leave this to a future enhancement to discuss